libs/dh/shared/feature-authorization/src/lib/scope-storage.service.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InjectionToken } from '@angular/core';
+
+export class ScopeStorage {
+  private length = 0;
+
+  public readonly getLength = () => this.length;
+
+  public readonly setItem = (key: string, value: string) => {
+    localStorage.setItem(key, value);
```

Comment:
Yes, it is safe, because:
- Scopes are not secret, they are just an API ressource, `api://<guid>/actor.default`
- Scopes do not grant permission, tokens do. If you ask for a token using a scope you are not part of, you will get nothing.
- If I "steal" a scope from localstorage than belongs to another user, I would have a random guid representing an actor. I would still need to request an access_token using this scope - that would fail if I am not a member of that actor.

We do not want to clear the scopes at logout, as we store the users preferred scope/actor across sessions.